indexer/src/main/java/org/pdxfinder/dataloaders/updog/domainobjectcreation/DomainObjectCreator.java

```diff
@@ -104,9 +104,9 @@ void createProvider(Map<String, Table> pdxDataTables) {
         log.info("Creating provider");
         Table finderRelatedTable = pdxDataTables.get("metadata-loader.tsv");
         Row row = finderRelatedTable.row(0);
-        String providerName = row.getString(TSV.Metadata.name.name());
-        String abbrev = row.getString(TSV.Metadata.abbreviation.name());
-        String internalUrl = row.getString(TSV.Metadata.internal_url.name());
+        String providerName = getCellAsText(row,TSV.Metadata.name.name());
```

Comment:
I am not sure how strict we are in terms of style, but would you consider to add a space after the commas? I think it makes it easier to read

Reply:
I have not been using a style formatter. So, I would expect a lot of inconsistencies like this. I like the space after this too. I can make a commit using the google style guide